tests/modules/ecs6-class/line.test.js

```diff
@@ -6,37 +6,31 @@ describe('Line constructor', () => {
         const point1 = new Point({ x: 1, y: 1 });
         const point2 = new Point({ x: 3, y: 3 });
         const line = new Line({ point1, point2 });
-
         expect(line.point1).toBe(point1);
         expect(line.point2).toBe(point2);
         expect(line.slope).toBe(undefined);
         expect(line.n).toBe(undefined);
     });
 });
-
 describe('Line calculateSlope method', () => {
     it('should correctly calculate the slope between point1 and point2', () => {
         const point1 = new Point({ x: 1, y: 1 });
         const point2 = new Point({ x: 3, y: 3 });
         const line = new Line({ point1, point2 });
         line.calculateSlope();
-
         expect(line.slope).toBe(1);
     });
 });
-
 describe('Line calculateNOfLineFunction method', () => {
     it('should correctly calculate the n value', () => {
         const point1 = new Point({ x: 1, y: 1 });
         const point2 = new Point({ x: 3, y: 3 });
         const line = new Line({ point1, point2 });
         line.calculateSlope();
         line.calculateNOfLineFunction();
-
```

Comment:
it the tests, you test only one function.
Why do you call the calculateSlope here?